static/src/javascripts-legacy/projects/common/modules/experiments/tests/paid-card-logo.js

```diff
@@ -0,0 +1,54 @@
+define([
+    'bean',
+    'qwery',
+    'lib/$',
+    'lib/config'
+], function (
+    bean,
+    qwery,
+    $,
+    config
+) {
+    return function () {
+        this.id = 'PaidCardLogo';
+        this.start = '2017-03-22';
```

Comment:
I'm going to have to edit some config before we start the test, so I'll double check it all then 👍